Cargo.toml

```diff
@@ -32,3 +33,8 @@ default = ["ansi_term", "nu-ansi-term"]
 name = "lscolors"
 path = "src/bin.rs"
 required-features = ["ansi_term", "nu-ansi-term"]
+
+[profile.release]
+lto = true
```

Comment:
Is this actually effective for lib crates?  I assume the downstream bin crate actually controls the LTO config

Reply:
Note that this is in the `[[bin]]` section. We ship a small executable with this crate. LTO/stripping is here mainly for consistency with some of my other crates. Probably not very relevant for this particular executable.

Reply:
Ahh right.  If GitHub had shown just one more line of context I would have seen that :)